notebooks/text_classification/text_classification_unified_information_explainer.ipynb

```diff
@@ -4,8 +4,6 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
```

Comment:
is removing the Microsoft copyrights ok from CELA?

Reply:
We should check this to be sure and follow the convention accordingly/document it somewhere

Reply:
yes it is as these were copied from the Azure Machine Learning notebooks, but are no longer needed in the OSS repo